src/hangman/secret.clj

```diff
@@ -33,11 +33,21 @@
   [secret]
   (str/join (map #(if (:visible %) (:char %) \_) secret)))
 
+(defn secret-string-initial
```

Comment:
This function is almost the same as the one above, and the docstring is also copied..
Maybe better to add a flag to the previous function?